src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -1184,6 +1184,23 @@ public void groupByScriptedHistogram() throws Exception {
         Assert.assertThat(result, containsString("\"script\":{\"source\""));
     }
 
+    @Test
+    public void distinctWithOneField() {
```

Comment:
what is the behavior if I want to run an aliased query (e.g SELECT DISTINCT name.lastname AS alias FROM ...)

Reply:
It works exactly the same with the proper GROUP BY, for example in the JDBC format there’s a row of “alias”:  “alias”